cypress.config.js

```diff
@@ -0,0 +1,7 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+  },
+});
```

Comment:
```suggestion
});

```

Recomenda-se deixar uma linha em branco ao final de todos os arquivos.

Isso faz com que ferramentas como o GitHub não exibam o ícone de `No newline at end of file` durante o processo de _code review_.

Além disso, isso é uma convensão e boa prática a ser seguida.

Reply:
Considere o mesmo para outros arquivos que "ferem" esta regra.